src/shadowbox/README.md

```diff
@@ -1,203 +1,169 @@
 
-Set a data limit for all access keys
-(e.g. limit outbound data transfer access keys to 1MB over 30 days)
+### Manual
 
-```
-curl -v --insecure -X PUT -H "Content-Type: application/json" -d '{"limit": {"bytes": 1000}}' $API_URL/experimental/access-key-data-limit
-```
+1. Prerequisites
 
-Remove the access key data limit
+   - A locally built Docker image containing your modifications.
+   - Your Docker image uploaded to a registry (e.g., Docker Hub, Quay.io).
 
-```
-curl -v --insecure -X DELETE $API_URL/experimental/access-key-data-limit
-```
+1. Setup
 
-## Testing
+   - **Environment Variable:** Set `SB_IMAGE` to the uploaded image location. Example:
 
-### Manual
+     ```sh
+     export SB_IMAGE=yourdockerhubusername/shadowbox
+     ```
 
-After building a docker image with some local changes,
-upload it to your favorite registry
-(e.g. Docker Hub, quay.io, etc.).
+   - **Start the Server:**
 
-Then set your `SB_IMAGE` environment variable to point to the image you just
-uploaded (e.g. `export SB_IMAGE=yourdockerhubusername/shadowbox`) and
-run `npm run action server_manager/electron_app/start` and your droplet should be created with your
-modified image.
+     ```sh
+     npm run action shadowbox/docker/start
+     ```
 
 ### Automated
 
-To run the integration test:
-
-```
-npm run action shadowbox/integration_test/run
-```
+1. **Build and Run:**
 
-This will set up three containers and two networks:
+   ```sh
+   npm run action shadowbox/integration_test/run
+   ```
 
-```
-client <-> shadowbox <-> target
-```
+   This does the following:
 
-`client` can only access `target` via shadowbox. We create a user on `shadowbox` then connect using the Shadowsocks client.
+   - Sets up three containers (`client`, `shadowbox`, `target`) and two networks.
+   - Creates a user on `shadowbox`.
+   - Connects to `target` through `shadowbox` using a Shadowsocks `client`: `client <-> shadowbox <-> target`
 
-To test clients that rely on fetching a docker image from Dockerhub, you can push an image to your account and modify the
-client to use your image. To push your own image:
+1. **Testing Client Modifications:**
 
-```
-npm run action shadowbox/docker/build && docker tag quay.io/outline/shadowbox $USER/shadowbox && docker push $USER/shadowbox
-```
+   - **Build Your Image:**
 
-If you need to test an unsigned image (e.g. your dev one):
+     ```sh
+     npm run action shadowbox/docker/build && docker tag quay.io/outline/shadowbox $USER/shadowbox && docker push $USER/shadowbox
+     ```
 
-```
-DOCKER_CONTENT_TRUST=0 SB_IMAGE=$USER/shadowbox npm run action shadowbox/integration_test/run
-```
+   - **Test Unsigned Images:** For development images, disable Docker contant trust:
 
-You can add tags if you need different versions in different clients.
+     ```sh
+     DOCKER_CONTENT_TRUST=0 SB_IMAGE=$USER/shadowbox npm run action shadowbox/integration_test/run
+     ```
 
-### Testing Changes to the Server Config
+1. **Testing Server Config Changes:**
```

Comment:
I think the old title was clearer. You are also deleteing the explanation of why you would do that.

Reply:
The title just seems longer while not adding anything distinct, but sure reverting.

The explanation of why you would do it is still there ("If your changes modify server startup config, remove the old test config: cmd"), just merged into the command bullet so it's less verbose and easier to parse. But I don't feel strongly about any particular statement (overall I think the README was just suffering from being overtly verbose and having a lot of prose so I'm trying to condense as much as possible to help the reader), so I can revert this one.